geomstats/geometry/hyperbolic.py

```diff
@@ -834,3 +879,38 @@ def dist(self, point_a, point_b):
         else:
             raise NotImplementedError(
                 'dist is only implemented for ball and extrinsic')
+
+    def retraction(self, tangent_vec, base_point):
+        """Poincaré ball model retraction.
+
+        Approximate the exponential map of hyperbolic space,
+        currently working only with poincare ball.
+        The retraction used is the one proposed in "Poincaré
```

Comment:
Use standard syntax for adding references in the docstring (see docstrings in stiefel.py for example).